rosidl_typesupport_c/cmake/rosidl_typesupport_c_generate_interfaces.cmake

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(NOT rosidl_generator_c_FOUND)
+  message(FATAL_ERROR
+    "Executing rosidl_typesupport_c extension point when rosidl_generator_c wasn't found."
```

Comment:
Nitpick: this string needs to have a trailing space to not have the second sentence follow the period without.

I also find the message kind of hard to read, maybe swap the order:

> 'rosidl_generator_c' not found when executing 'rosidl_typesupport_c' extension.

Reply:
Done, see https://github.com/ros2/rosidl_typesupport/pull/73/commits/5d01aa94fad084f5b9f2316a3fbd3e42f8e91c3c.